certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -589,13 +589,88 @@
         "x-ms-visibility": "important"
       }
     },
-    "/accounts/{accountId}/envelopes": {
+    "/accounts/{accountId}/envelopes/createFromTemplateNoRecipients": {
```

Comment:
Good catch. Surprisingly, it works without it. Maybe in the eSignature code they are using path prefix instead of exact path when matching for end point. But to be consistent, I will rewrite it.